.github/workflows/mac.yml

```diff
@@ -121,7 +121,7 @@ jobs:
           NIGHTLY_REPO_TOKEN: ${{ secrets.NIGHTLY_REPO_TOKEN }}
           NIGHTLY_REPO: ${{ github.repository_owner }}/servo-nightly-builds
       - name: Build package for target
-        run: gtar -czf target.tar.gz target/${{ inputs.profile }}/servo target/${{ inputs.profile }}/*.dylib target/${{ inputs.profile }}/lib/*.dylib resources
+        run: gtar -czf target.tar.gz target/${{ inputs.profile }}/servo target/${{ inputs.profile }}/lib/*.dylib resources
```

Comment:
Please remove this change line since it's no longer needed after the rebase.

Reply:
I do not think so: https://github.com/servo/servo/blob/d10688b5ef3dceb2b8a0e5ca4e089cba12bca0cd/.github/workflows/mac.yml#L139

target/${{ inputs.profile }}/*.dylib is only matched with script_plugin.dylib and now that it does not exist this step fails.